php/php.editor/test/unit/src/org/netbeans/modules/php/editor/verification/ImplementAbstractMethodsHintErrorTest.java

```diff
@@ -169,6 +170,8 @@ public void testUnionTypes_04() throws Exception {
         checkHints(new ImplementAbstractMethodsHintError(), "testUnionTypesImplementMethod04.php");
     }
 
+    // fails on Windows
```

Comment:
I am not sure how this test could fail on Windows and pass on Linux... it is an editor test, so it is OS independent. Or am I missing something?

Reply:
I thought it was a due to the line endings, but changing the test file to use unix line ending still resulted in the following failure. https://github.com/oyarzun/netbeans/runs/5449321610?check_suite_focus=true#step:20:7291

```[junit] Testcase: testStaticReturnType_01(org.netbeans.modules.php.editor.verification.ImplementAbstractMethodsHintErrorTest):	FAILED
[junit] Content does not match between 'testfiles/verification/ImplementAbstractMethodsHintError/testStaticReturnTypeImplementMethod01.php' and 'testfiles/verification/ImplementAbstractMethodsHintError/testStaticReturnTypeImplementMethod01.phptestStaticReturnType_01.hints'
[junit] Expected content is:
[junit] 
[junit] class TestClass implements TestInterface {
[junit]       ---------
[junit] HINT:\TestClass is not abstract and does not override abstract method  testSimple() in \TestInterface
[junit] FIX:Implement All Abstract Methods
[junit] FIX:Declare Abstract Class
[junit] 
[junit] but actual is:
[junit] 
[junit] class TestClass implements TestInterface {
[junit]       ---------
[junit] HINT:\TestClass is not abstract and does not override abstract method  testNullableType() in \TestInterface
[junit] FIX:Implement All Abstract Methods
[junit] FIX:Declare Abstract Class
[junit] 
[junit] It differs in the following things:
[junit] 
[junit] Actual content contains following lines which are missing in expected content: 
[junit] 	HINT:\TestClass is not abstract and does not override abstract method  testNullableType() in \TestInterface
[junit] Expected content contains following lines which are missing in actual content: 
[junit] 	HINT:\TestClass is not abstract and does not override abstract method  testSimple() in \TestInterface
```

Reply:
I see, thanks for clarification. Perhaps @junichi11 would know here?

Reply:
Maybe the cause is `Set`. I'll try to improve it later (tomorrow or later).

Reply:
@oyarzun Fixed: https://github.com/apache/netbeans/pull/3734